qiita_client/qiita_client.py

```diff
@@ -215,7 +215,17 @@ def _request_oauth2(self, req, *args, **kwargs):
             kwargs['headers']['Authorization'] = 'Bearer %s' % self._token
         else:
             kwargs['headers'] = {'Authorization': 'Bearer %s' % self._token}
-        r = req(*args, **kwargs)
+
+        # in case the Qiita server is not reachable (workers are busy), let's
+        # try for 3 times with a 30 sec sleep between tries
+        retries = 3
+        while retries > 0:
```

Comment:
It looks like if retries reaches zero that this code silently returns without ever issuing a req?

Reply:
Should an exception be raised if retries is 0 and no successful req was made?

Reply:
excellent point, fixing ...